### DISCUSSION TITLE
s/7g2wncQG1lIziOmtBxXwEw

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#71] [Announcements]
# s/7g2wncQG1lIziOmtBxXwEw

每个Geek对折腾自己的博客都有着一份执念

https://blog.ops-coffee.cn/s/7g2wncQG1lIziOmtBxXwEw

**Comment by zhaoyansheng163** [comment]
请教两个问题：
1、部署之后，默认账号密码是多少啊 admin/admin不好使。是在哪里配置的啊？
2、去掉登录页面之后（#@login_required(login_url='/login')），可以直接进入主页，点击detail会报错
no such table: ops_coffee_blog

**Reply by ops-coffee** [reply]
是我的疏忽，我以为大伙都懂django

建表语句：

python manage.py makemigrations
python manage.py migrate

创建账号：

python manage.py createsuperuser
